src/quo2/components/selectors/reactions/view.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo2.components.selectors.reactions.view
+  (:require [quo2.components.reactions.resource :as reactions.resource]
+            [quo2.components.selectors.reactions.style :as style]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]))
+
+(defn view
+  [_ {:keys [start-pressed?]}]
+  (let [pressed? (reagent/atom start-pressed?)]
+    (fn [emoji
+         {:keys [container-style on-press
+                 accessibility-label]
+          :or   {accessibility-label :reaction}}]
+      [rn/touchable-without-feedback
+       {:accessibility-label accessibility-label
+        :on-press            (fn [e]
+                               (swap! pressed? not)
+                               (when on-press
+                                 (on-press e)))}
+       [rn/view
+        {:test-ID (if @pressed? :pressed :released)
```

Comment:
should we use an accessibility label instead?

Reply:
I opted for a test-ID because the accessibility ID is already used in the touchable above, and this one would have no purpose except for the component spec. I can change to `accessibility-id` though :)

Reply:
Why do we need to expose the pressed state of a view through `:test-ID`?
I think if we want to test if something has been pressed, we can use a mock callback and some RTL functions.

Probably this is not the best use of `:test-ID` :thinking:  wdyt?

Reply:
FYI: So in the end, after talking to @ulisesmac, I removed these assertions because they're testing an internal implementation.

Reply:
FYI: So in the end, after talking to @ulisesmac, I removed these assertions because they're testing an internal implementation.